Tensile/SolutionStructs.py

```diff
@@ -2980,20 +2980,27 @@ def assignDerivedParameters(state):
         reject(state, "int8 doesn't support LocalSplitU")
         return
 
+    # GlobalSplitU doesn't work with some other things:
     if state["GlobalSplitU"] > 1:
-        # GlobalSplitU doesn't work with some other things:
-        if not state["GlobalSplitUSummationAssignmentRoundRobin"] and state["LoopTail"]:
-          reject(state, "GlobalSplitU and LoopTail require SummationAssignmentRoundRobin=True since strongly breaks Tensile kernel architecture")
-          return
+      if not state["GlobalSplitUSummationAssignmentRoundRobin"] and state["LoopTail"]:
+        reject(state, "GlobalSplitU and LoopTail require SummationAssignmentRoundRobin=True since strongly breaks Tensile kernel architecture")
+        return
 
-        # GSU is working for all gemm functions, except for I8II.
-        if (state["ProblemType"]["DataType"].isInt8() or state["ProblemType"]["DataType"].isInt8x4()):  
-          reject(state, "GlobalSplitU is not for I8 or 4xi8 data type.")
-          return
+      supported = \
+        (state["ProblemType"]["DataType"].isSingle()) or \
+        (state["ProblemType"]["DataType"].isDouble() and state["BufferStore"]) or \
```

Comment:
Why only double type needs BufferStore?

Reply:
Not supported, Some tests failed with BF0.

Reply:
My question is why only double?
What is the difference between single and double?